documentation/Pure DP Mechanism.lyx

```diff
@@ -0,0 +1,652 @@
+
+ and 
+\begin_inset Formula $f$
+\end_inset
+
+ into the field using 
+\begin_inset Formula $\pi_{\mathbb{F}_{p}}$
+\end_inset
+
+.
+ The un-noised aggregate shares produced by Prio3 are secret shares of the query result,
+ 
+\begin_inset Formula $q^{*}=q^{\left(0\right)}+q^{\left(1\right)}$
+\end_inset
+
+.
+ Each aggregator samples noise from the discrete Laplace distribution and adds it to the un-noised aggregate shares,
+ and then sends the sum as their aggregate share to the collector.
+ If we pessimistically assume that only one honest aggregator out of the two aggregators is adding differential privacy noise,
+ then the mechanism produces 
+\begin_inset Formula $M\left(x\right)=q^{\left(0\right)}+q^{\left(1\right)}+\pi_{\mathbb{F}_{p}}\left(Z\right)=q^{*}+\pi_{\mathbb{F}_{p}}\left(Z\right)$
+\end_inset
+
+,
+ where 
+\begin_inset Formula $Z_{j}\leftarrow\mathrm{Lap}_{\mathbb{Z}}\left(\nicefrac{GS_{f}}{\varepsilon}\right)$
+\end_inset
+
+ is drawn independently for all 
+\begin_inset Formula $1\leq j\le d$
+\end_inset
+
+.
+\end_layout
+
+\begin_layout Theorem
+\begin_inset Formula $M\left(x\right)=\pi_{\mathbb{F}_{p}}\left(f\left(x\right)\right)+\pi_{\mathbb{F}_{p}}\left(Z\right),Z_{j}\leftarrow Lap_{\mathbb{Z}}\left(\nicefrac{GS_{f}}{\varepsilon}\right)$
+\end_inset
+
+ satisfies 
+\begin_inset Formula $\varepsilon$
+\end_inset
+
+-differential privacy.
+\end_layout
+
+\begin_layout Proof
+We will show Definition 1 holds for singleton events,
+ where 
+\begin_inset Formula $E$
+\end_inset
+
+ is a set of cardinality one,
+ then other events will follow by a union bound.
+\end_layout
+
+\begin_layout Proof
+Let 
+\begin_inset Formula $q=f\left(x\right)$
+\end_inset
+
+,
+ 
+\begin_inset Formula $q^{\prime}=f\left(x^{\prime}\right)$
+\end_inset
+
+,
+ and 
+\begin_inset Formula $q^{*}=\vec{\pi}_{\mathbb{F}_{p}}\left(f\left(x\right)\right)$
+\end_inset
+
+,
+ and let 
+\begin_inset Formula $q_{j}$
+\end_inset
+
+,
+ 
+\begin_inset Formula $q_{j}^{*}$
+\end_inset
+
+,
+ and 
+\begin_inset Formula $Z_{j}$
+\end_inset
+
+ denote the 
+\begin_inset Formula $j$
+\end_inset
+
+-th component of the respective vectors.
+ Then 
+\begin_inset Formula $M_{j}\left(x\right)=q_{j}^{*}+\pi_{\mathbb{F}_{p}}\left(Z_{j}\right)$
+\end_inset
+
+.
+ Applying the probability density function of the discrete Laplace distribution,
+ we have:
+\begin_inset Formula 
+\[
+\forall j\in\left[d\right],y_{j}\in\mathbb{F}_{p},\mathbb{P}\left[M_{j}\left(x\right)=y_{j}\right]=\mathbb{P}\left[\pi_{\mathbb{F}_{p}}\left(Z_{j}\right)=y_{j}-q_{j}^{*}\right]
+\]
+
+\end_inset
+
+
+\begin_inset Formula 
+\[
+=\stackrel[k=-\infty]{\infty}{\sum}\mathbb{P}\left[Z_{j}=\pi_{\mathbb{Z}}\left(y_{j}\right)-q_{j}+kp\right]
+\]
+
+\end_inset
+
+
+\begin_inset Formula 
+\[
+=\stackrel[k=-\infty]{\infty}{\sum}\frac{e^{\nicefrac{\varepsilon}{GS_{f}}}-1}{e^{\nicefrac{\varepsilon}{GS_{f}}}+1}e^{\nicefrac{-\varepsilon\left|\pi_{\mathbb{Z}}\left(y_{j}\right)-q_{j}+kp\right|}{GS_{f}}}
+\]
```

Comment:
A common alternative to $e^X$ is $\exp(X)$. I think it would make this easier to read.